core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerFacade.java

```diff
@@ -343,4 +343,18 @@ void detach(AdvertView advertView) {
             advertView.detach(adsLoader);
         }
     }
+
+    void disableAdverts() {
+        if (adsLoader.isPresent()) {
+            NoPlayerAdsLoader adsLoader = this.adsLoader.get();
```

Comment:
is `this` needed?

Reply:
yes, because the `adsLoader` on the left is a local variable and the `this` on the right is a field.

Reply:
oh I see, be careful on that, it might be better to just rename the variables so that there is no need for `.this`